docs/source/models.rst

```diff
@@ -2443,6 +2444,8 @@ expected input to the model to ensure your inference request can be read properl
 \**** The mask syntax for the model that you've chosen is going to be specific to that model's implementation. Some are '[MASK]', while others are '<mask>'. Verify the expected syntax to
 avoid failed inference requests.
 
+\***** If using MLServer for realtime inference, a raw audio file in bytes format must be base64 encoded prior to submitting to the endpoint.
```

Comment:
Does this only apply to MLServer (https://pypi.org/project/mlserver-mlflow/ - Seldon), or does it apply to the MLflow Model Server in general?

Reply:
uhhh whoops. Serving in general. chalk this up to writing this docstring while in a meeting that someone brought up MLServer.